scripts/clean.js

```diff
@@ -0,0 +1,33 @@
+// This script is run from package.json in the root of the project before
```

Comment:
Usually [rimraf](https://www.npmjs.com/package/rimraf) is used for this purpose (cross OS file removal)

Reply:
I just lifted the upx script. I've deleted it and changed the `package.json` to use `rimraf` now